.bazelversion

```diff
@@ -1,4 +1,4 @@
-6.1.1
```

Comment:
latest bazel releases have fixes for coverage when running with "build without the bytes"